draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -654,476 +526,437 @@
-   RECOMMENDED that Trigger Status Resources are not automatically
-   deleted by dCDN for at least 24 hours after they become
-   "complete", "processed", "failed", or "cancelled".</t>
-
-   <t>
-   To ensure that it can get the status of its Trigger Status
-   Resources for completed and failed CI/T Commands, it is RECOMMENDED
-   that uCDN polling interval is less than the time after which
-   records for completed activity will be deleted.</t>
 
    </section>
 
-   <section title="Loop Detection and Prevention" anchor="loop-detection-and-prevention"><t>
-   Given three CDNs, A, B, and C, if CDNs B and C delegate delivery of
-   CDN A's content to each other, CDN A's CI/T Commands could be passed
-   between CDNs B and C in a loop. More complex networks of CDNs could
-   contain similar loops involving more hops.</t>
-
-   <t>
-   To prevent and detect such CI/T loops, each CDN uses a CDN
-   Provider ID (PID) to uniquely identify itself. In every CI/T Command
-   it originates or cascades, each CDN MUST append an array element
-   containing its CDN PID to a JSON array under an entry named
-   "cdn-path". When receiving CI/T Commands, dCDN MUST check the
-   cdn-path and reject any CI/T Command that already contains its own
-   CDN PID in the cdn-path. Transit CDNs MUST check the cdn-path and
-   not cascade the CI/T Command to dCDNs that are already listed in the
-   cdn-path.</t>
-
-   <t>
-   The CDN PID consists of the two characters "AS" followed by the CDN
-   provider's Autonomous System number <xref target="RFC1930"/>, then a colon (":") and
-   an additional qualifier that is used to guarantee uniqueness in case
-   a particular AS has multiple independent CDNs deployed -- for
-   example, "AS64496:0".</t>
-
-   <t>
-   If the CDN provider has multiple ASes, the same AS number SHOULD be
-   used in all messages from that CDN provider, unless there are
-   multiple distinct CDNs.</t>
-
+   <section title="Error Handling" anchor="error-handling">
    <t>
-   If the CDNI Request Routing Redirection interface (RI) described in
-   <xref target="RFC7975"/> is implemented by dCDN, the CI/T interface and the RI
-   SHOULD use the same CDN PID.</t>
-
-   </section>
-        
-   <section title="Error Handling" anchor="error-handling"><t>
-   dCDN can signal rejection of a CI/T Command using HTTP status codes
-   -- for example, 400 ("Bad Request") if the request is malformed, or
-   403 ("Forbidden") or 404 ("Not Found") if uCDN does not have
-   permission to issue CI/T Commands or it is trying to act on another
-   CDN's data.</t>
+   	dCDN MAY reject CI/T interface requests, by responding to those requests with 4xx or 5xx HTTP status codes.
+  	For example, uCDN MAY respond with 400 ("Bad Request") if the request is malformed, or 403 ("Forbidden") 
+	or 404 ("Not Found") if the request could not be properly authenticated or if uCDN is trying to act on another CDN's resources
```

Comment:
Missing period at the end of the sentence.

Reply:
fixed